extensions/scarb-doc/tests/data/json_doc_hidden_impls.json

```diff
@@ -70,7 +70,7 @@
               "item_data": {
                 "name": "SecondVisibleImpl",
                 "doc": null,
-                "signature": "impl SecondVisibleImpl of SecondVisibleTrait<HiddenStruct, VisibleStruct>",
+                "signature": "impl SecondVisibleImpl of SecondVisibleTrait<<missing>, VisibleStruct>;",
```

Comment:
i will check and fix this with https://github.com/software-mansion/scarb/issues/2070